### PR TITLE
Docs Update from open-telemetry/opentelemetry-collector

### PR DESCRIPTION
**Pull request by austinlparker**
This is an automated pull request.

### CODE REVIEW

**Comment by austinlparker** [comment]
There was some drift that needs to be synced back to the collector repo before this gets merged

**Comment by austinlparker** [comment]
These changes have been synced in https://github.com/open-telemetry/opentelemetry-collector/pull/3267, this PR can be merged or we can wait for another push.

File: content/en/docs/collector/getting-started.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ $ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
     cd opentelemetry-collector/examples; \
     go build main.go; ./main & pid1="$!";
     docker run --rm -p 13133:13133 -p 14250:14250 -p 14268:14268 \
-      -p 55678-55680:55678-55680 -p 8888:8888 -p 9411:9411 \
+      -p 55678-55679:55678-55679 -p 4317:4317 -p 8888:8888 -p 9411:9411 \
       -v "${PWD}/otel-local-config.yaml":/otel-local-config.yaml \
```
Comment (mxiamxia) [inline content/en/docs/collector/getting-started.md:79]:
```suggestion
      -v "${PWD}/local/otel-config.yaml":/otel-local-config.yaml \
```

**Comment by austinlparker** [comment]
@mxiamxia if you'd like to make changes to this, you need to make a PR against the authoritative copy in the collector repo (https://github.com/open-telemetry/opentelemetry-collector/tree/main/website_docs) so they don't drift out of sync

**Comment by mxiamxia** [comment]
> @mxiamxia if you'd like to make changes to this, you need to make a PR against the authoritative copy in the collector repo (https://github.com/open-telemetry/opentelemetry-collector/tree/main/website_docs) so they don't drift out of sync

OK. l'll send a separate PR to correct the docker starting command. Fine with merging this PR.

**Review by mxiamxia — APPROVED** [review]
LGTM

**Pull request closed without merging**